MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -635,6 +635,13 @@ private StreamInfo BuildVideoItem(MediaSourceInfo item, MediaOptions options)
             var isEligibleForDirectStream = options.EnableDirectStream && (options.ForceDirectStream || !bitrateLimitExceeded);
             TranscodeReason transcodeReasons = 0;
 
+            // Force transcode or remux for BD/DVD folders
+            if (item.VideoType == VideoType.Dvd || item.VideoType == VideoType.BluRay)
+            {
+                isEligibleForDirectPlay = false;
+                isEligibleForDirectStream = false;
```

Comment:
Is there a reason to limit DirectStream? Ffmpeg will still spit out a file/repackage into a file using DirectStream, right?

Reply:
That's correct. I removed that line since it works without it too. Doesn't change what's put out though.